### DISCUSSION TITLE
Can i change the start day of the week in calendar without changing the locale?

### DISCUSSION THREAD

**Discussion by VladSez** [adobe/react-spectrum#3553] [Q&A]
I want the week to start on Monday, is it possible?
[Codesandbox](https://codesandbox.io/s/dank-bash-8nsp0z?file=/src/Calendar.js)

![image](https://user-images.githubusercontent.com/16666234/192162927-4fc5b910-3354-476b-920f-bcc7af9079a4.png)

**Comment by reidbarber** [comment]
~You can just reorder the returned days like:~
```
  let [first, ...rest] = weekDays;
  weekDays = [...rest, first];
```

~Example: https://codesandbox.io/s/calendar-starting-on-monday-q6kr3c?file=/src/CalendarGrid.js~

---

Update: there is an issue for this at https://github.com/adobe/react-spectrum/issues/3986

**Reply by VladSez** [reply]
super! thank you

**Reply by ArtemFedorchuk** [reply]
But in this way, you will see no correct dates
Example: 
<img width="407" alt="Screenshot 2023-07-05 at 12 44 19" src="https://github.com/adobe/react-spectrum/assets/43110065/ac57d1bb-21dd-46ea-b3ae-845f629cedb7">
Today is Wednesday the 5 of July, but the calendar shows us Thursday as the current day.

Have you a solution for it?

**Reply by tuananhlai** [reply]
This is not really a solution, since it only change the headers and not the dates themselves.

**Reply by VladSez** [reply]
cc @reidbarber

**Reply by reidbarber** [reply]
Unfortunately there isn't currently a solution. There is an open issue with some more discussion here: https://github.com/adobe/react-spectrum/issues/3986
